roles/distro-update/tasks/main.yml

```diff
@@ -1,16 +1,19 @@
 ---
 
-- name: APT
+- name: APT update cache as separate step
   apt:
     update_cache: yes
+  when: ansible_pkg_mgr == "apt"
+  failed_when: false
```

Comment:
Can you explain this line?

Reply:
Not really, it was there before, I just converted this role to YAML dictionary format. We should probably remove, though this PR has been merged already.